c7n/mu.py

```diff
@@ -47,6 +47,7 @@
 log = logging.getLogger('custodian.serverless')
 
 LambdaRetry = get_retry(('InsufficientPermissionsException',), max_attempts=2)
+RuleRetry = get_retry(('ResourceNotFoundException',), max_attempts=5)
```

Comment:
this should be 2, there's back off already between attempts

Reply:
Thanks!